functions/auth/processSignUp.js

```diff
@@ -1,7 +1,12 @@
 const functions = require("firebase-functions");
 const admin = require("firebase-admin");
 
-const emailWhitelist = ["jpangs88@gmail.com", "susannebjorkblom@hotmail.com"];
+const emailWhitelist = [
+  "jpangs88@gmail.com",
+  "susannebjorkblom@hotmail.com",
+  "lndpangalos@gmail.com",
+  "iris.pangalos@gmail.com"
+];
```

Comment:
Would add `seanmheff@gmail.com` in here too

Reply:
hahahaahhaha good catch